README.md

```diff
@@ -13,5 +13,7 @@ Version 6.0.0 is introducing dozens of breaking changes, mostly through updated
 - [Documentation](API.md)
 - [Changelog](CHANGELOG.md)
 - [Contribution guide](CONTRIBUTING.md)
+- [Usefull Side Project](./docs/PROJECTS.md)
```

Comment:
Perhaps such a link could be useful in [API.md at L428](https://github.com/react-native-video/react-native-video/blob/master/API.md?plain=1#L428)?

Reply:
OK, done, thank you for the catch !